xcm/src/v0/multi_location.rs

```diff
@@ -56,6 +56,14 @@ pub enum MultiLocation {
 	X3(Junction, Junction, Junction),
 	/// A relative path comprising four junctions.
 	X4(Junction, Junction, Junction, Junction),
+	/// A relative path comprising four junctions.
```

Comment:
looks like we need some macro to generate those?

Reply:
yeah, that'd be nice

Reply:
https://github.com/paritytech/polkadot/issues/2750